test-cases/longevity/longevity-large-partition-3h.yaml

```diff
@@ -40,3 +40,5 @@ nemesis_interval: 15
 user_prefix: 'longevity-large-partitions-3h'
 
 space_node_threshold: 644245094
+
+run_fullscan: 'scylla_bench.test, 10, allow_reversed_queries' # 'ks.cf|random, interval(min)''
```

Comment:
Why did you add it only to this longevity?
Please add to all large partitions longevities.

Now the problem is that it’s random and we can’t control it like this.
So if one want to test this feature he needs to wait and hope to get it running enough times.
Please change it so we can control it.